lib/recurly/base.php

```diff
@@ -194,8 +194,10 @@ public function getLinks() {
     'plan_code' => 'string',
     'plan_codes' => 'array',
     'pending_subscription' => 'Recurly_Subscription',
+    'risk_rules_triggered' => 'array',
     'redemption' => 'Recurly_CouponRedemption',
     'redemptions' => 'Recurly_CouponRedemptionList',
+    'rule' => 'Recurly_FraudRiskRule',
```

Comment:
I think you'd need to use some class type just because of the way the values are assigned. So we could probably do `stdClass` but I think a named class is better form.